--- src/app/actions.ts
+++ src/app/actions.ts
@@ -2,2 +2,3 @@
 
+import { unstable_noStore as noStore } from "next/cache";
 import prisma from "@/lib/prisma";
@@ -93,2 +94,3 @@
 export async function getHero() {
+  noStore();
   try {
@@ -103,2 +105,3 @@
 export async function getAbout() {
+  noStore();
   try {
@@ -113,2 +116,3 @@
 export async function getContact() {
+  noStore();
   try {
@@ -123,2 +127,3 @@
 export async function getSocialLinks() {
+  noStore();
   try {
@@ -133,2 +138,3 @@
 export async function getPortfolioCategories() {
+  noStore();
   try {
@@ -143,2 +149,3 @@
 export async function getPortfolios() {
+  noStore();
   try {
@@ -153,2 +160,3 @@
 export async function getPackages() {
+  noStore();
   try {
@@ -162,2 +170,3 @@
 export async function getFaqs() {
+  noStore();
   try {
